src/sentry/relay/config/__init__.py

```diff
@@ -200,6 +207,11 @@ def _get_project_config(project, full_config=True, project_keys=None):
 
             cfg["config"]["dynamicSampling"] = {"rules": active_rules}
 
+    # In this case we should override old conditionnal rules if they exists
+    # or just return uniform rule
+    if allow_dynamic_sampling_basic:
+        cfg["config"]["dynamicSampling"] = {"rules": [generate_uniform_rule(project)]}
```

Comment:
do we want to capture_exception here?
E.g. if 
```
try:
    cfg["config"]["dynamicSampling"] = {"rules": [generate_uniform_rule(project)]}
except Exception:
        sentry_sdk.capture_exception()
```
Especially when `quotas.get_blended_sample_rate(project)` returns None?

Reply:
That's definitely the safest option. Though if `get_blended_sample_rate` returns `None`, there won't be an exception, will there? Best to handle that case separately.

Reply:
yeah it's mostly question, cause we discussed wit @ahmedetefy to make it required and may be tweak function/ to raise an exception if it's None.

Reply:
I agree with @jjbayer here, I think we should handle that separately in `generate_uniform_rule` and just not return dynamic sampling config to relay